### PR TITLE
[FEATURE] Add description to manuals on Guides & Tutorials

### PR DESCRIPTION
**Pull request by sypets**
- add an additional column "category"
- add descriptions for all manuals
- change column "Comment" to "Description"

### CODE REVIEW

**Comment by sypets** [comment]
It would be beneficial to decrease the width of the second column but that can be considered later.

**Comment by DanielSiepmann** [comment]
I've updated some descriptions. Just check out the commit.
If everything is fine for you, hit the merge button.

File: Documentation/typo3cms/GuidesAndTutorials/Index.rst [hunk 1]
```diff
@@ -77,18 +103,29 @@ represents a logical progression in the knowledge of the product.
  :header-rows: 1
```
Comment (DanielSiepmann) [inline Documentation/typo3cms/GuidesAndTutorials/Index.rst]: Should we move the tutorials before the guides?
They are more important to newcomers. Experienced users might find the guides even if they are more "hidden".

Comment (sypets) [reply Documentation/typo3cms/GuidesAndTutorials/Index.rst]: @DanielSiepmann I agree to move the Tutorials before the Guides. I had the same thought. I think, someone else already suggested the same thing (can't remember who it was).

Thank you for your help!

I would suggest merging first and then to make further changes, but it seems you are doing it that way already with the other PR.

Comment (DanielSiepmann) [reply Documentation/typo3cms/GuidesAndTutorials/Index.rst]: I've pushed the change.

**Pull request merged**